resty/ipmatcher.lua

```diff
@@ -27,6 +27,10 @@ local AF_INET     = 2
 local AF_INET6    = 10
 if ffi.os == "OSX" then
     AF_INET6 = 30
+elseif ffi.os == "BSD" then
+    AF_INET6 = 28
```

Comment:
Could you give a comment about the source of the magic number?

Reply:
The links below contain the AF_INET6 values for BSD like os and Windows

FreeBSD: https://docs.freebsd.org/en/books/developers-handbook/sockets/

Windows: https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-socket

Lua ffi.os(jit.os) only defines  "Windows", "Linux", "OSX", "BSD", "POSIX" or "Other"  as target OS. 

So add bsd like os and windows support should cover most user cases.